examples/postgresql-ephemeral-template.json

```diff
@@ -245,8 +245,8 @@
     {
       "name": "POSTGRESQL_VERSION",
       "displayName": "Version of PostgreSQL Image",
-      "description": "Version of PostgreSQL image to be used (9.2, 9.4, 9.5 or latest).",
-      "value": "9.5",
+      "description": "Version of PostgreSQL image to be used (9.4, 9.5, 9.6 or latest).",
```

Comment:
Is 9.2 not relevant anymore?

Reply:
Has been deprecated for a long while already.

Reply:
Ok. We'll need to keep an eye for that.

Reply:
pg_isready is not available for 9.2 anyway